plenum/test/replica/test_buffers_cleaning.py

```diff
@@ -0,0 +1,63 @@
+from plenum.server.replica import Replica
+
+
+class FakeNode():
```

Comment:
But this replica functionality will happen after all the initialisation and a running node